src/components/App.css

```diff
@@ -0,0 +1,138 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
+body {
+  background-color: red;
+  background-image: url("../imgs/bg.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 0;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  padding: 0 5vw;
+}
+
+/* Home page */
+
+.header {
+  text-align: center;
+  font-size: 20vh;
+  margin: 0;
+}
+
+.ul-links {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+}
+
+.li-links {
+  font-size: 8vh;
+  font-weight: bold;
+  color: darkblue;
+  padding: 0 20px;
+}
```

Comment:
- [ ] kindly note that one of the requirements says that The general layout should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).  Kindly make sure this general layout(Navbar, calculator) is matched in your design.


![image](https://user-images.githubusercontent.com/65068771/159791335-fc237ecd-ae71-485e-8057-6dd108473c9b.png)

Reply:
Yes, you are right.
This is sad, though. I put a lot of effort in trying a different design...
But I'll change it to match the requirements.